charts/camunda-platform/values.yaml

```diff
@@ -1746,6 +1746,105 @@ web-modeler:
       # Postgresql.auth.database defines the name of the database to be created for Web Modeler
       database: web-modeler
 
+# Connectors configuration for the Connectors.
+connectors:
+  # Enabled if true, the Connectors deployment and its related resources are deployed via a helm release
+  enabled: true
+
+  # Switch for inbound mode (e.g., for webhook or polling)
+  inbound:
+    # Acceptable values: disabled, credentials, or oauth
```

Comment:
We need to add a constraint for this to avoid having typos.

here is an example [constrains.tpl](https://github.com/camunda-community-hub/camunda-7-community-helm/blob/main/charts/camunda-bpm-platform/templates/constrains.tpl)
use [mustHas](https://helm.sh/docs/chart_template_guide/function_list/#has-musthas) function to make sure the value of the mode is one of the acceptable values.

Reply:
What is the issue you see? :-)

Reply:
Do you use the same Helm version as in the repo?
https://github.com/camunda/camunda-platform-helm/blob/e3e1c3d27eb6d7cacfd3368e76663abce3089bc6/.tool-versions#L1

With that version, it works as expected locally (we can just change `mustHas` to `has` since we already handle the error with `fail` but it has nothing to do with functionally, both work as expected)

Reply:
Seems that I have the newer. But I am afraid that customers might also use different Helm version.

```
~$ helm version
version.BuildInfo{Version:"v3.11.1", GitCommit:"293b50c65d4d56187cd4e2f390f0ada46b4c4737", GitTreeState:"clean", GoVersion:"go1.19.5"}
```

Since it works for you, I will just change `mustHas` -> `has`. Thanks for confirming!